log-insights/setup/google-cloud-sql/index.mdx

```diff
@@ -1,102 +0,0 @@
----
```

Comment:
In the interest of not breaking links, and to have a proper hierarchy, should we keep these `index.md` pages, but just link to the individual steps underneath, instead of having any text on them?

Reply:
Sure. I didn't think it was worth the maintenance, but these will probably stay pretty static.